master/pkg/schemas/expconf/experiment_config.go

```diff
@@ -29,7 +29,7 @@ type ExperimentConfigV0 struct {
 	RawEnvironment              *EnvironmentConfigV0        `json:"environment"`
 	RawHyperparameters          HyperparametersV0           `json:"hyperparameters"`
 	RawLabels                   LabelsV0                    `json:"labels"`
-	RawLogPolicies              LogPoliciesConfigV0         `json:"log_policies"`
+	RawLogPolicies              *LogPoliciesConfigV0        `json:"log_policies"`
```

Comment:
why change this to pointers to a slice throughout the PR?

Reply:
Good point! A slice doesn't need a pointer.